### PR TITLE
Blog post about comparison `0.0` and `-0.0`

### LINKED ISSUE #43: Blog post about comparison `0.0` and `-0.0`
In EO we have the next results of comparison `0.0` and `-0.0`:
```
0.0.eq -0.0 # FALSE
-0.0.eq 0.0 # FALSE
0.0.gt -0.0 # FALSE
-0.0.gt 0.0 # FALSE
0.0.lt -0.0 # FALSE
-0.0.lt 0.0 # FALSE
0.0.gte -0.0 # FALSE
-0.0.gte 0.0 # FALSE
0.0.lte -0.0 # FALSE
-0.0.lte 0.0 # FALSE
```
So they are not incomparable at all. It seems that such situation is unique in comparison with other programming languages. 
It would be nice to make a little experiment how it works in some other programming languages (python, java, c++, ruby, ...) and write a short blog post about it.

### PR DESCRIPTION
**Pull request by c71n93**
Closes #43

---

### Detailed summary:

- Added a new blog post titled "Comparison of 0.0 and -0.0"
- Explained the differences between `0.0` and `-0.0` in number encodings
- Described the previous comparison behavior in EO and how it differed from other languages
- Discussed how `0.0` and `-0.0` are considered equal in other programming languages
- Explained the changes made to the `eq` attribute in EO to comply with the IEEE 754 standard

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by c71n93** [comment]
@maxonfjvipon could you check this blog post, please. I left the section `Conclusion` unfinished, because I don't know the answer to the questions I left there yet.

**Comment by c71n93** [comment]
@maxonfjvipon could you check this, please?

File: _posts/2023/10/2023-10-20-comparison-of-floating-zeros.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+comparison operators"* (referencing the standard in the blog post isn't possible as it's a paid document, but you can 
+find this information [here](https://en.wikipedia.org/wiki/Signed_zero)).
+
+Thus, this behavior for float numbers is built into most CPUs and works at the assembler level. Comparison operators, 
+such as `==`, in the languages discussed above, correspond to how it is implemented in the hardware. This is indeed the 
+case in most languages.
+
+### Comparison of 0.0 and -0.0 in EO now
+
+In order for the floating-point comparison in EO to comply with the standard, we have changed the behavior of the `eq`
```
Comment (maxonfjvipon) [inline _posts/2023/10/2023-10-20-comparison-of-floating-zeros.md:69]: @c71n93 I would add here that we didn't introduce atoms for `eq` and still compare floats bitwise but added special cases for zeros

**Review by maxonfjvipon — CHANGES_REQUESTED** [review]
@c71n93 Good one! I like it, just a few comments from my side

**Comment by c71n93** [comment]
@maxonfjvipon I resolved your comments, could you check please?

**Review by maxonfjvipon — CHANGES_REQUESTED** [review]
@c71n93 please change date in the name of the file and I think it good to go

**Comment by c71n93** [comment]
@maxonfjvipon done

**Review by maxonfjvipon — APPROVED** [review]
@c71n93 good one, thanks

**Comment by c71n93** [comment]
@yegor256 could you check this one, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**